feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -131,6 +131,17 @@ func TestInterfaceOperStatus(t *testing.T) {
 	}
 }
 
+func TestInterfacePhysicalChannel(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	dp := dut.Port(t, "port1")
+
+	phyChannel := dut.Telemetry().Interface(dp.Name()).PhysicalChannel().Get(t)
+	t.Logf("Got %s PhysicalChannel from telmetry: %v", dp.Name(), phyChannel)
```

Comment:
Got PhysicalChannel %q from telemetry. TODO for me to add a pretty print utility in ygot for GoStructs.

Reply:
Done